src/sentry/south_migrations/0367_auto__chg_field_release_ref__chg_field_release_version.py

```diff
@@ -0,0 +1,980 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
As discussed, flip this back to True and I'll just run the SQL by hand to make sure so I can respond quicker if there were an issue.